models/evc.py

```diff
@@ -171,6 +170,31 @@ def sync(self, keys: set = None):
             return
         self._mongo_controller.upsert_evc(self.as_dict())
 
+    def _get_unis(self, **kwargs) -> (UNI, UNI):
+        """Obtain both UNIs (uni_a, uni_z).
+        If a UNI is changing, verify tags"""
+        uni_a = kwargs.get("uni_a", None)
+        uni_a_flag = False
+        if uni_a and uni_a != self.uni_a:
+            uni_a_flag = True
+            self._use_uni_vlan(uni_a)
```

Comment:
Shouldn't this also be in a try catch block too since it can raise `ValuError`? 

Also, the method is OK, but notice the name is a bit deceptive, it's doing what more than only getting UNIs, maybe you could rename it as `_get_unis_use_tags` or something else closely related? Great docstring.

Reply:
Added catching `ValueError` and re-raising error. I haven't changed the name though, keeping an eye.

Reply:
Changed name to `_get_unis_use_tags()`